block/block.go

```diff
@@ -60,19 +60,19 @@ const (
 )
 
 const (
-	Initiator_NLSS_share   string = "nlss_share_signature"
-	Initiator_Private_sign string = "priv_signature"
-	Initiator_DID          string = "initiator_did"
-	Initiator_Hash         string = "hash"
-	Initiator_SignType     string = "sign_type"
+	InitiatorNLSSShare   string = "nlss_share_signature"
+	InitiatorPrivateSign string = "priv_signature"
+	InitiatorDID         string = "InitiatorDID"
```

Comment:
Any specific reason the string value is "InitiatorDID" and not "initiatorDID"

Reply:
It's a typo. I'll modify this one in some other branch, as this PR is closed.

Reply:
No worries